src/lists.py

```diff
@@ -42,17 +45,29 @@ def print_list(x):
 match sys.argv[1]:
     case "mean":
         # put your solution to the first exercise here
-        mean = "mean of x"
+
+        #calculate the sum 
+        sum = 0
+        for i in x:
+            sum += i
+        #calculate the mean
+        
+        mean = (sum/len(x))
         print(mean)
 
     case "times":
         # Put your solution to the second exercise here
         times_three = []
+        for i in x:
+            times_three.append(i*3)
         print_list(times_three)
 
     case "even":
         # Put your solution to the third exercise here
         even = []
+        for j in x:
+            if (j % 2 == 0):
+                even.append(j)
```

Comment:
Perfect.

And an excuse to learn more about list comprehensions :)

List comprehensions can also have an `if` part, and you could write this code as

```python
even = [j for j in x if j % 2 == 0]
```